modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,134 @@ It also includes enhancements to the Eventing and Search services.
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-601]
+== Release 6.0.1
+
+Couchbase Server 6.0.1, released in February 2019, is the first maintenance release in the 6.0.x series for Couchbase Server.
+
+[#supported-platforms-601]
+=== New Supported Platforms
+
+This release adds support for Amazon Linux 2 and Ubuntu 18.04 LTS platforms.
+
+See xref:install:install-platforms.adoc[Supported Platforms] for the complete list of supported platforms.
+
+[#fixed-issues-601]
+=== Fixed Issues
+
+The notable fixes in this release are listed below.
+
+==== Analytics Service
+
+[#table_fixedissues_v601-analytics,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-32435[MB-32435^]
+| *Summary*: Fixed an intermittent rebalance failure observed when upgrading the Analytics node.
```

Comment:
Read the comments from Murtadha Hubail on 04/Jan/19 9:14 PM for accurate description of the issue seen in 6.0

Reply:
Thanks! Here's the updated description:
"Fixed an issue where a rebalance operation could fail due to the reuse of dataset IDs internally."